src/js/pages/HowItWorks.jsx

```diff
@@ -256,6 +258,7 @@ class HowItWorks extends Component {
   }
 
   handleChangeSlide = (selectedStepIndex) => {
+    this.sendHowItWorksSlideEvent(selectedStepIndex);
```

Comment:
Let's look up the titleId and pass that in instead of the StepIndex. That way it is a human readable (and searchable) id like "claimYourCampaignProfile". You will need to pass into handleChangeSlide the 'currentSlides', so you can convert the `selectedStepIndex` to the titleId.